uwblueprint/childrens-aid-society
# Set up design system text styles

## Notion ticket link

[Set up design system text styles](https://www.notion.so/uwblueprintexecs/Set-up-design-system-text-styles-64afe790818643a69d4e6bfd93f3ba7d)



## Implementation description
* Set up design system text styles



## Steps to test
1. Check diff text styles in `/` (homepage screen
2. Check out other screens and make sure the styling looks okay
![image](https://user-images.githubusercontent.com/32009013/180658302-82caef6b-d63a-4a81-be56-bb921f9aba4c.png)



## What should reviewers focus on?
* do the styles look alright? 


## Checklist
